docs/release/supported_platforms.md

```diff
@@ -14,8 +14,9 @@ OSG also supports select rebuilds of RHEL.  Specifically:
 -   Scientific Linux 7
 
 OSG builds and tests its RPMs on the latest releases of the relevant platforms (e.g., in 2018, the RHEL 7 builds were based on RHEL 7.5).
-Older releases of the OS may not receive thorough testing and have subtle bugs; we may ask sites to update to the latest OS packages
-as part of the support process.
+Older releases of the OS may not receive thorough testing and have subtle bugs.
```

Comment:
```suggestion
Older releases of the OS may not receive thorough testing and may have subtle bugs.
```

Reply:
Older releases of the OSG may not receive thorough testing, may have subtle bugs, and damn well have obvious bugs. :smiley: